compiler/src/lsp/lenses.re

```diff
@@ -0,0 +1,76 @@
+open Grain_typed;
+
+type lens_t = {
+  line: int,
+  signature: string,
+};
+
+let lens_to_yojson = (l: lens_t): Yojson.t =>
+  `Assoc([("line", `Int(l.line)), ("signature", `String(l.signature))]);
+
+let lens_list_to_yojson = (lenses: list(lens_t)): Yojson.t =>
+  `List(List.map(l => lens_to_yojson(l), lenses));
+
+let lenses_to_json_string = (l: list(lens_t)): string =>
+  Yojson.to_string(lens_list_to_yojson(l));
```

Comment:
Could you just `[@deriving yojson]` the `lens_t` here?

Reply:
Now I know, yes I can :-). And a much better solution!